### ISSUE TITLE
Visual Studio 2015 support

### ISSUE THREAD

**Issue by saltares** [SonyWWS/LevelEditor#26]
* The renderer doesn't load properly when using the VS2010 project.
* When using the VS2013 project, you need to upgrade to use the VS2015 compiler.
* `hash_set` and `hash_map` are deprecated (we should be using `unordered_set` and `unordered_map` instead. 

Do you accept PRs to include a VS2015 solution and use the non deprecated classes?

**Comment by abeckus** [comment]
Hi David,
Thanks for pointing out the issue.
I will replace deprecated map/set soon.
Regarding vs2015, I think this will come soon too. 

Alan

**Comment by saltares** [comment]
Thanks for the reply! Do you recommend having a separate local branch with these fixes and wait for your changes or would you prefer me to submit a pull request?

**Comment by abeckus** [comment]
Yes, I think it would be better to have local copy for now, and wait for the changes.
Because I don't have any plan to add new features before vs2015 support.
The plan is to remove vs2010 and add vs2015 while keeping vs2013.

Alan

**Comment by saltares** [comment]
Sounds good to me!
